### PR TITLE
Changes for the new release and nightlies

### PR DESCRIPTION
**Pull request by jmcarcell**
New versions, don't compile Geant4 with VecGeom and add debug symbols for some repositorios plus some latest versions.

### CODE REVIEW

File: environments/key4hep-common/packages.yaml [hunk 1]
```diff
@@ -12,13 +12,17 @@ packages:
   gdb:
     require: +python+tui+source-highlight~debuginfod
   # https://github.com/key4hep/key4hep-spack/pull/501
+  geant4:
+    require: +qt+opengl
```
Comment (andresailer) [inline environments/key4hep-common/packages.yaml:16]: Why not be explicit about the no vecgeom here? or does `~vecgeom` not work?

Comment (jmcarcell) [reply environments/key4hep-common/packages.yaml:16]: Well it has been the default forever, I don't think it matters for this one

Comment (andresailer) [reply environments/key4hep-common/packages.yaml:16]: What is "it" that was the default? 

Can we be explicit if anything ever requires vecgeom in geant for whatever reason, so we get a conflict? Now that we still remember this issue?

Comment (jmcarcell) [reply environments/key4hep-common/packages.yaml]: Done

Comment (andresailer) [reply environments/key4hep-common/packages.yaml]: Thanks!

**Comment by jmcarcell** [comment]
Release built with these settings; nothing left here

**Pull request merged**